src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Text.Json
 {
-    public enum DuplicatePropertyNameHandling
+    public enum DuplicatePropertyNameHandlingStrategy
```

Comment:
Why did we change this? We have other enums within options (like `JsonCommentHandling`), which don't append "Strategy" as a suffix.

Reply:
Yes, that's why I wanted to change it somehow not to have two things being named the same.  But probably there's a better naming to chose.

Reply:
> Yes, that's why I wanted to change it somehow not to have two things being named the same.

Sounds good.

> But probably there's a better naming to chose.

Right, that can be deferred for now.